src/Symfony/Component/Form/FormFactoryInterface.php

```diff
@@ -23,7 +23,8 @@ interface FormFactoryInterface
      *
      * @see createBuilder()
      *
-     * @param mixed $data The initial data
+     * @param class-string<FormTypeInterface>|'Symfony\Component\Form\Extension\Core\Type\FormType' $type
```

Comment:
is the part about `FormType` really needed? what does it provide? can we remove it?

Reply:
this change looks wrong to me. `class-string<FormTypeInterface>` really means `FormTypeInterface::class`, not "class name of a class extending FormTypeInterface"

Reply:
I took this from the psalm stubs and what @stof says makes sense. I had to make it a union for it to accept the default value.

But yeh I doubt it adds much for the DX and is a headache to get right. I will remove this.